pkg/cluster/cluster.go

```diff
@@ -415,6 +422,11 @@ func (c *Cluster) startSeedMember(recoverFromBackup bool) error {
 		SecureClient: c.isSecureClient(),
 	}
 	ms := etcdutil.NewMemberSet(m)
+	if c.IsPodPVEnabled() {
+		if err := c.createPVC(m); err != nil {
```

Comment:
This is duplicated in two places. Can you combine it with createPod() into a new function? Maybe createResource?